.mci.yml

```
@@ -1236,6 +1234,13 @@ tasks:
         - func: "run_kms_servers"
         - func: "test"
 
+    - name: docker_build_arm
+      run_on: &docker-distros
```

```suggestion
      run_on:
```

Suggest removing unused YAML anchor `&docker-distros` here and in the `docker_build` task.